crates/nu-command/tests/commands/reduce.rs

```diff
@@ -46,35 +46,6 @@ fn reduce_rows_example() {
     assert_eq!(actual.out, "14.8");
 }
 
-#[test]
-fn reduce_numbered_example() {
-    let actual = nu!(
-        cwd: ".", pipeline(
-        r#"
-        echo one longest three bar
-        | reduce -n { |it, acc| if ($it.item | str length) > ($acc.item | str length) {echo $it} else {echo $acc}}
-        | get index
-        "#
-        )
-    );
-
-    assert_eq!(actual.out, "1");
-}
```

Comment:
Can we keep this test but in the new format? I can't really figure out how to make it work right now.
This prints `longest` as it should but I'm not sure how to get the index.
```
echo one longest three bar | reduce {|it, acc, idx| if ($it | str length) > ($acc | str length) { $it } else { $acc }}
```